src/block_cache/block_chunkcache.c

```diff
@@ -412,7 +412,7 @@ __chunkcache_eviction_thread(void *arg)
                     WT_STAT_CONN_INCR(session, chunk_cache_chunks_evicted);
                     __wt_verbose(session, WT_VERB_CHUNKCACHE,
                       "evicted chunk: %s(%u), offset=%" PRId64 ", size=%" PRIu64,
-                      (char *)&chunk->hash_id.objectname, chunk->hash_id.objectid,
+                      (char *)chunk->hash_id.objectname, chunk->hash_id.objectid,
```

Comment:
Good spot. That said, it shouldn't need the `char *` cast. In fact, I think this was technically undefined behaviour previously since the `char *` cast casts away the `const`. This also happens on lines 258 and 716, would you be able to fix it up there as well please?

Reply:
Update [473f1d9](https://github.com/wiredtiger/wiredtiger/pull/9650/commits/473f1d9dcbc2223f3b1ebb8d49c8034b57845e08)

Reply:
Updated [402e60b](https://github.com/wiredtiger/wiredtiger/pull/9650/commits/402e60b6f302f81ce47d2c2f9b864a742437dfc9)